io.catenax.secondary_material_content/RELEASE_NOTES.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 All notable changes to this model will be documented in this file.
 
-## [Unreleased]
+## [Deprecated]
```

Comment:
i think better would be to remove this 'Unreleased' tag to avoid any confusion. it is not important to keep it.